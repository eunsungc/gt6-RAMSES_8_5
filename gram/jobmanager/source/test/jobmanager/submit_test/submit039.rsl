&(executable = /bin/echo)
 (arguments  = "single quotes shouldn't break things")
