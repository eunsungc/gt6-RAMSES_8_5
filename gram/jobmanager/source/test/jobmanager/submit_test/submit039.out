single quotes shouldn't break things
